src/pages/About.scss

```diff
@@ -94,3 +94,7 @@
   }
 
 }
+
+#date-input-popover {
+  --offset-y: -var(--ion-safe-area-bottom);
```

Comment:
```suggestion
  --offset-y: var(--ion-safe-area-bottom);
```

Reply:
Removing the negative causes the offset to stop working, and the calendar is still cut off.
![Screenshot 2021-11-08 141359](https://user-images.githubusercontent.com/90629384/140811395-b5931cf6-3fd0-439f-81e9-cbe8b9d16c6b.png)

Reply:
Oh 🤦  yes you are correct. I thought that was a stray `-`. Nevermind!